swarm/storage/hasherstore.go

```diff
@@ -172,6 +175,7 @@ func (p *hasherStore) encryptChunkData(chunkData ChunkData) (ChunkData, encrypti
 }
 
 func (h *hasherStore) decryptChunkData(chunkData ChunkData, encryptionKey encryption.Key) (ChunkData, error) {
+	log.Info("decryptChunkData", "length", len(chunkData))
```

Comment:
This should def not be an `Info`, but rather a `Trace` or `Debug`. I'd say a `Trace`.